src/SmartComponents/ActivityTable/Columns.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import propTypes from 'prop-types';
 import { renderColumnComponent } from '../../Utilities/helpers';
+import { Icon } from '@patternfly/react-core/dist/dynamic/components/Icon';
```

Comment:
I was not true. While migrating remediation app, I found out direct path is encouraged for optimisation.

Reply:
What @mkholjuraev first said is actually correct and we should only have to use `'@patternfly/react-core'` and not a direct path. 

This will require to disable the (eslint) rule that actually rewrites the path.